### PR TITLE
sms_to_twitter: move keys to config

### PR DESCRIPTION
**Pull request by tenzap**

### CODE REVIEW

File: application/plugins/sms_to_twitter/config/sms_to_twitter.php [hunk 1]
```diff
@@ -12,3 +12,7 @@
 */
 // phpcs:enable
 $config['twitter_code'] = 'TW';
+
+// Fill with your OAuth credentials
+$config['consumer_key'] = '';
```
Comment (kingster) [inline application/plugins/sms_to_twitter/config/sms_to_twitter.php:17]: Populate default with existing keys maybe?

Comment (kingster) [reply application/plugins/sms_to_twitter/config/sms_to_twitter.php:17]: We should update this in release notes in that case, that people who wants this plugin needs to generate their own keys in this case.

Comment (tenzap) [reply application/plugins/sms_to_twitter/config/sms_to_twitter.php:17]: Maybe I don't understand what these keys are for. Is it normal that all the world can send to this twitter account? I thought these are private. What would be the purpose of such credentials if it is world writable?

Comment (kingster) [reply application/plugins/sms_to_twitter/config/sms_to_twitter.php:17]: Couldn't find docs of this type of keys on twitter, probably this plugin needs to be updated with the newer twitter api's.

**Pull request merged**